packages/@o3r/styling/project.json

```
@@ -14,7 +14,17 @@
       "options": {
         "project": "packages/@o3r/styling/ng-package.json",
         "tsConfig": "packages/@o3r/styling/tsconfig.build.json"
-      }
+      },
+      "inputs": [
+        "source",
+        "^source",
+        "{projectRoot}/schemas/*.json",
+        "{projectRoot}/scss/**/*",
+        "!{projectRoot}/scss/**/*.spec.ts",
+        "{projectRoot}/assets/**",
+        "{projectRoot}/_index.scss",
+        "{projectRoot}/_otter-theme.scss"
```

```suggestion
        "source",
        "^source",
        "schemas",
        "{projectRoot}/scss/**/*.scss",
        "{projectRoot}/assets/**",
        "{projectRoot}/_index.scss",
        "{projectRoot}/_otter-theme.scss"
```

(maybe it could be interesting to add a `namedInput` for `{projectRoot}/assets/**` as it is used in several places)

---

I will do this in a separate PR